swift-sdk/Internal/InAppHelper.swift

```diff
@@ -375,12 +377,33 @@ struct InAppHelper {
             channelName: channelName,
             messageId: messageId,
             campaignId: campaignId,
+            trigger: trigger,
             edgeInsets: edgeInsets,
             backgroundAlpha: backgroundAlpha,
             html: html,
             extraInfo: extraInfo))
     }
     
+    private static func parseTrigger(fromTriggerElement element: [AnyHashable : Any]?) -> IterableInAppTriggerType {
+        guard let element = element else {
+            return .immediate
+        }
+        guard let triggerTypeString = element[.ITBL_IN_APP_TRIGGER_TYPE] as? String else {
+            return .immediate
+        }
+        
+        switch triggerTypeString.lowercased() {
+        case String(describing: IterableInAppTriggerType.immediate).lowercased():
+            return .immediate
+        case String(describing: IterableInAppTriggerType.event).lowercased():
+            return .event
+        case String(describing: IterableInAppTriggerType.never).lowercased():
+            return .never
+        default:
+            return .immediate
```

Comment:
I think it's better to default to `.never`

Reply:
Agreed. We should open a Jira for this.